templates/web/base/admin/emergencymessage/index.html

```diff
@@ -11,12 +11,16 @@
     <tbody>
         [% FOR body IN bodies %]
         [% emergency_message = body.get_extra_metadata('emergency_message') %]
+        [% emergency_message_waste = body.get_extra_metadata('emergency_message_waste') %]
         <tr>
             <td>[% body.name %]</td>
-            <td>[% IF emergency_message %][% emergency_message.substr(0, 60) %]&hellip;[% END %]</td>
+            <td>
+                [% IF emergency_message %]<p>[% emergency_message.substr(0, 60) %]&hellip;[% END %]
+                [% IF emergency_message_waste %]<p>[% emergency_message_waste.substr(0, 60) %]&hellip;[% END %]
```

Comment:
Possibly there should be something here to distinguish between these?

Reply:
This page is only for superusers, so I wasn't sure I could be bothered, but you're right, it should.